numpy/core/fromnumeric.py

```diff
@@ -380,6 +380,7 @@ def choose(a, choices, out=None, mode='raise'):
     See Also
     --------
     ndarray.choose : equivalent method
+    numpy.take_along_axis : link:https://docs.scipy.org/doc/numpy-1.15.1/reference/generated/numpy.take_along_axis.html
```

Comment:
If truly needed, this should be simply `take_along_axis` with no `numpy` prefix and no `: <link>` since the link is created automagically by sphinx. 

Why did you specify `take_along_axis` and not `take` and general indexing (which are ways to index a single ndarray)? 

`choose` is a method to combine values from multiple arrays, why should they cross-reference methods for a single array?